index.js

```diff
@@ -1,25 +1,21 @@
 const thize = (i) => {
-  i = Number(i) 
+  i = Number(i);
```

Comment:
Does it make sense to count negative numbers though?

Reply:
Will add it anyway

Reply:
Lol it doesn't make sense. But just in case, I think it make sense for negative numbers to have the same th/st/nd/rd as their positive numbers.

i.e.

```
thize(x) = thize(-x)
```